src/Payload.php

```diff
@@ -341,11 +341,7 @@ public function setUrlArgs(array $value): Payload
      */
     public function setCustomValue(string $key, $value): Payload
     {
-        if ($key === self::PAYLOAD_ROOT_KEY) {
-            throw InvalidPayloadException::reservedKey();
-        }
-
-        $this->customValues[$key] = $value;
+        $this->customValues = array_merge_recursive($this->customValues ? $this->customValues : [], [$key => $value]);
```

Comment:
Why don't we need here the check - if `$key === self::PAYLOAD_ROOT_KEY` ?

Reply:
Because now it's not removes previously defined data in PAYLOAD_ROOT_KEY, it's merges.

In my code I do the following:
public function toApnVoip($notifiable)
    {
        return ApnVoipMessage::create()
            ->custom('aps', [
                'alert' => [
                    'incoming_caller_id' => (string)$this->card->card_id,
                    'incoming_caller_name' => $this->card->therapist->fullName,
                    'uuid' => (string) \Str::uuid(),
                    'card' => $this->card
                ]
            ])
            ->badge(1);
    }
    
    So I set 'aps' as custom array, but I dont removes preciously setted data

Reply:
I think it should be a separate method. Something like addCustomValue, not setCustomValue.

Reply:
May be you can just remove check that field named 'aps' ?